flink-runtime/src/test/java/org/apache/flink/runtime/executiongraph/utils/SimpleAckingTaskManagerGateway.java

```diff
@@ -89,13 +89,12 @@ public String getAddress() {
 	}
 
 	@Override
-	public CompletableFuture<StackTraceSampleResponse> requestStackTraceSample(
-			ExecutionAttemptID executionAttemptID,
-			int sampleId,
-			int numSamples,
-			Time delayBetweenSamples,
-			int maxStackTraceDepth,
-			Time timeout) {
+	public CompletableFuture<TaskBackPressureSampleResponse> sampleTaskBackPressure(
```

Comment:
`sampleTaskBackPressure`  -> `requestTaskBackPressure` ? also not necessary for final arguments change.